build.sh

```diff
@@ -18,11 +18,8 @@ HANDLERS="cloudwatch-handler s3-handler sns-handler mysql-handler postgresql-han
 
 for HANDLER in ${HANDLERS}; do
 	cd ${HANDLER}
-	GOOS=linux go build -ldflags "-X github.com/honeycombio/agentless-integrations-for-aws/common.version=${VERSION}"
+	GOARCH=arm64 GOOS=linux go build -ldflags "-X github.com/honeycombio/agentless-integrations-for-aws/common.version=${VERSION}" -o bootstrap
```

Comment:
Love that you noticed the lack of ARM64 support! I know this makes it slightly more complex, but I think we should likely build and package _both_ architectures.

Reply:
ah yeah that's a good point! let me add that in